tests/udp.rs

```diff
@@ -0,0 +1,79 @@
+use anyhow::*;
+use clap::Parser;
+use mavlink_server::{cli, drivers, hub, logger, stats, web};
+use serial_test::serial;
+use tracing::*;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 10)]
+async fn test_udpserver_receive_only() -> Result<()> {
+    cli::init_with(cli::Args::parse_from(vec![
+        "", // For some unknown reason, the first argument is ignored
+        "udpclient:0.0.0.0:3333",
+        "udpserver://0.0.0.0:3333?direction=receiver",
+        "--mavlink-heartbeat-frequency",
+        "10",
+    ]));
+
+    for driver in cli::endpoints() {
+        hub::add_driver(driver).await?;
+    }
+
+    stats::set_period(tokio::time::Duration::from_millis(100)).await;
+    tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
+    for (uuid, driver) in stats::drivers_stats().await.unwrap() {
+        if *driver.name == "UdpServer" {
+            assert!(driver.stats.output.is_none());
+            assert!(driver.stats.input.is_some());
+        }
+
+        if *driver.name == "UdpClient" {
+            assert!(driver.stats.output.is_some());
+            assert!(driver.stats.input.is_none());
+        }
+    }
+
+    for (id, driver_info) in hub::drivers().await? {
+        debug!("Removing driver id {id:?} ({driver_info:?})");
+        hub::remove_driver(id).await?;
+    }
+
+    Ok(())
+}
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 10)]
+#[ignore]
```

Comment:
why ignore this?

Reply:
The tests can't run in parallel, since will conflict with each other, I did running them serialized, but no luck, we need to improve this in the future.

Reply:
Hmm, I see. We have the same issue on the mavlink camera manager, but there the serial test works. This is one sad downside of using singletons :(